### PR TITLE
Products

### PR DESCRIPTION
**Pull request by purvasharmadev**
## Major changes:
 - Added the featured product component using filter and map.
 - Integrated the mockman for API testing

## Minor Changes
 - Created routes for landingPage and mockman
 - added the stylesheets for product
 - added product data

featured-product

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,12 +1,23 @@
...
+import {LandingPage} from "./Pages/LandingPage";
 
 function App() {
+
+
+
   return (
     <div>
+      <Routes>
+      <Route path="/mock-api" element={<MockAPI />} />
```
Comment (iRohitGaur) [inline src/App.js:17]: suggestion: use extentions in vscode (like prettier) for auto indentation

File: src/Components/Products/featured-product.jsx [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+    const [featuredProducts, setFeaturedProducts] = useState([])
+
+    // useState for loader
+    const [loader, setLoader] = useState(true)
+
+    // Api call
+    async function getProducts() {
+        try {
+          await axios.get("/api/products").then((res) => {
+            setFeaturedProducts(res.data.products);
```
Comment (iRohitGaur) [inline src/Components/Products/featured-product.jsx:19]: suggestion: check for response status

**Review by iRohitGaur — APPROVED** [review]
UI looks nice. Code is understandable. Good to merge 👍🏻

**Review by zoef-yt — APPROVED** [review]
Great work. The code is easy to understand. Looks good to merge.

File: src/App.js (hunk 1 above)
Comment (adityapandadev) [reply src/App.js:17]: Its a great extension, also bracket pair colorizer make life more simple

**Pull request merged**